resources/teamcity/includes/tc-helpers.inc.sh

```diff
@@ -66,3 +66,19 @@ set_variables_for_emscripten() {
   export EMSCRIPTEN_BASE="${EMSCRIPTEN_BASE:-${HOME}/emsdk/upstream/emscripten}"
   export KEYMAN_USE_EMSDK=1
 }
+
+upload_help() {
+  local PRODUCT=$1
+  local PRODUCT_PATH=$2
+  builder_echo start "upload help" "Uploading new ${PRODUCT} help to help.keyman.com"
+
+  export HELP_KEYMAN_COM="${HELP_KEYMAN_COM:-${KEYMAN_ROOT}/../help.keyman.com}"
```

Comment:
I would prefer that we don't have default fallbacks for build env variables -- it makes maintenance harder. Seems like HELP_KEYMAN_COM should always be already defined, making this line unnecessary.

Can be fixed in a future PR.